agent/server/server.go

```diff
@@ -126,6 +131,7 @@ type server struct {
 	tunnels               map[tunnelKey]*wg.Tunnel
 	tokens                *tokens.Tokens
 	cancelTokenMonitoring func()
+	cancelSynthetics      func()
```

Comment:
I don't see this ever getting called. Is that intentional?

Reply:
Nice catch! I forgot to remove it -- I was thinking to call the cancel function when user call `flyctl settings synthetics disable` but it was getting hairy and I stopped trying it in this version 😃 . Removed by now